api/grpc/types/api.pb.go

```diff
@@ -466,7 +466,7 @@ type UpdateResource struct {
 	CpuQuota                     uint64            `protobuf:"varint,4,opt,name=cpuQuota" json:"cpuQuota,omitempty"`
 	CpusetCpus                   string            `protobuf:"bytes,5,opt,name=cpusetCpus" json:"cpusetCpus,omitempty"`
 	CpusetMems                   string            `protobuf:"bytes,6,opt,name=cpusetMems" json:"cpusetMems,omitempty"`
-	MemoryLimit                  uint64            `protobuf:"varint,7,opt,name=memoryLimit" json:"memoryLimit,omitempty"`
+	MemoryLimit                  int64             `protobuf:"varint,7,opt,name=memoryLimit" json:"memoryLimit,omitempty"`
```

Comment:
You should change all memory fields which accept `-1` as unlimited by cgroup api.